server/routes/api/place.py

```diff
@@ -16,13 +16,21 @@
 import json
 import services.datacommons as dc
 
-from flask import Blueprint, request, jsonify
+from flask import Blueprint, jsonify, request, Response
 
 from cache import cache
 from services.datacommons import fetch_data
-from routes.api.stats import get_stats_wrapper
+import routes.api.stats as stats_api
 
-WANTED_PLACE_TYPES = [
+# Place types to keep for list of child places, keyed by parent place type.
+WANTED_PLACE_TYPES = {}
```

Comment:
can merge this into one statement?

WANTED_PLACE_TYPES = {
  'Country': [],
  'State': [],
  'County': []
}

Reply:
done